android-components/components/concept/awesomebar/src/main/java/mozilla/components/concept/awesomebar/AwesomeBar.kt

```diff
@@ -72,6 +72,17 @@ interface AwesomeBar {
      */
     fun setOnEditSuggestionListener(listener: (String) -> Unit)
 
+    /**
+     * Information about the [Suggestion]s that are currently displayed by the [AwesomeBar].
+     */
+    data class VisibilityState(
+        /**
+         * An ordered map of the currently visible [SuggestionProviderGroup]s, and the visible [Suggestion]s in each
+         * group. The groups are ordered top-to-bottom.
+         */
+        val groups: Map<SuggestionProviderGroup, List<Suggestion>> = emptyMap(),
```

Comment:
nit:
```suggestion
        val suggestionGroups: Map<SuggestionProviderGroup, List<Suggestion>> = emptyMap(),
```

or even

```suggestion
        val visibleSuggestionGroups: Map<SuggestionProviderGroup, List<Suggestion>> = emptyMap(),
```

Super pedantic, but `VisibilityState` implies to my brain that it could also account for `Invsibile` items in the future because old Android View's had `INVISIBLE`, `GONE`, and `VISIBLE` options for their visibility property so this language is kinda in the general lexicon.

Reply:
That makes sense; thanks for the additional context! 😊